pkg/shell/indexes.js

```diff
@@ -124,6 +125,66 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
             index.preload_frames(m, m.manifests);
     }
 
+    function selectActiveItem() {
+        const cur = document.activeElement;
+        if (cur.nodeName === "INPUT") {
+            const el = document.querySelector("#host-apps li:first-of-type > a");
+            if (el)
+                el.click();
+        } else {
+            cur.click();
+        }
+    }
+
+    function moveItem(nth_of_type, sibling) {
+        const cur = document.activeElement;
+        if (cur.nodeName === "INPUT") {
+            const item = document.querySelector("#host-apps li:" + nth_of_type + " > a");
+            if (item)
+                item.focus();
+        } else {
+            const next = cur.parentNode[sibling];
+            if (next)
+                next.children[0].focus();
```

Comment:
Same here -- the method focuses an element, what is actually being "moved" here?